_projects/2023/100481838/100481838.Rmd

```diff
@@ -0,0 +1,1067 @@
+
+This representation uses bell curves to compare the distribution in the frequency of occurrence of temperature anomalies (y-axis) divided by local standard deviation for the 1951-1980 period (x-axis) obtained by binning all local results for the indicated region and 11-year period into 0.05 frequency intervals. Additionally, this chart is presented along an animation which shows the transition of all recorded periods.
+
+![Note: This is a screen recording of the actual animation. I was unable to properly download it due to the way it was embedded in the NYT website. Apologies for the resolution.](Untitled%20video%20-%20Made%20with%20Clipchamp.gif)
+
+In essence, what this chart aims to communicate is the shift in the distribution of temperatures towards a higher occurrence of hotter temperatures. So, lets take a look at visual encoding it uses to do so...
+
+| [*REMINDER*]{.underline}: Visual encoding is the way in which data items and their attributes are mapped to visual structures such as marks and channels. Taking them From data items (observations, rows) to visual marks and from data attributes (variables, columns) to visual channels
+
+## Visual encoding
+
+Our raw data consists on rows representing anomalies in recorded temperatures ranging from -6 to +6 in 0.05 intervals. The attributes of these data include the frequency of occurrence of these anomalies and the period in which they were recorded. The visual elements used to map the data are as follow
+
+-   **Visual mark**: area - the shape of the distribution of occurrence in temperature anomalies
+
+-   **Visual channel (identity)**: color - determining which anomalies are to be regarded as extremely cold, cold, normal, hot or extremely hot along the X-axis.
+
+## Other elements
+
+In order to further the expressive capabilities of these chart, the original authors included further explanatory elements such as tittle and subtitle, text labels and a legend, as well as reference elements such as a reference area that allows comparison with the baseline period and reference lines which emphasize the limits of each temperature group.
+
+![Original chart from NYT website marked-up to list additional elements and annotations](marked%20up%20chart.png)
+
+# Replication
+
+## Original and tidy data
+
+The authors behind the publication that inspired this chart post data and visualizations of their own in this website <http://www.columbia.edu/~mhs119/PerceptionsAndDice/>
+
+From here, we download the 'shifted data' table named *Data: N.H. Land, Jun-Jul-Aug.* Which when imported to R is visualized as follows:
+
+```{r Import raw data, echo=TRUE}
+library(ggplot2)
+library(tidyverse)
+ccdata<- tibble(read.table(
+  file = "http://www.columbia.edu/~mhs119/PerceptionsAndDice/NH.JJA.11yrs.txt", 
+  header = TRUE,
+  skip = 1))
+
+head(ccdata)
```

Comment:
Please save the file as CSV, and read from the saved CSV instead of the remote URL, just in case it is not available in the future there. Also please remove the RData file.